jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/configuration/UploadResourceRunSteps.java

```diff
@@ -0,0 +1,84 @@
+package com.cerner.jwala.ui.selenium.steps.configuration;
+
+import com.cerner.jwala.ui.selenium.component.JwalaUi;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.By;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+/**
+ * Created by Rahul Sayini on 7/7/2017
+ */
+public class UploadResourceRunSteps {
+
+    @Autowired
+    private JwalaUi jwalaUi;
+
+    @Autowired
+    @Qualifier("parameterProperties")
+    private Properties paramProp;
+
+    @Given("^I am in the resource tab$")
+    public void goToConfigurationTab() {
```

Comment:
Nice catch, I just renamed this file from [Rahul's](https://github.com/cerner/jwala/blob/cae1c795a1c2d9a0927f62da8357706a562e4e7a/jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/configuration/ManageResourceRunSteps.java#L25). I'll rename it.